linux/datadog_wrapper

```diff
@@ -30,11 +30,14 @@ setUpCommonEnv() {
             rm trace-agent* 
         fi
 
-        echo "Downloading required datadog binaries from $DD_BINARIES_URL"
-        curl -L $DD_BINARIES_URL/$DD_AAS_LINUX_VERSION/$FILE -o $FILE || return
-
-        echo "Decompressing files"
-        tar -zxf $FILE || return
+        echo "Downloading required Datadog binaries from $DD_BINARIES_URL/$DD_AAS_LINUX_VERSION/$FILE"
+        if curl -L --fail $DD_BINARIES_URL/$DD_AAS_LINUX_VERSION/$FILE -o $FILE; then
+            echo "Decompressing files"
+            tar -zxf $FILE || return
+        else 
+            echo "Failed to download the Datadog binary succesfully."
```

Comment:
This line should return so we don't try to start the trace agent.

Reply:
Alternatively, you can add the `|| return` to the curl line and remove the if block all together.